tests/test_connection.py

```diff
@@ -2196,6 +2196,45 @@ def test_parse_transport_parameters(self):
         )
         client._parse_transport_parameters(data)
 
+    def test_parse_transport_parameters_idle_time(self):
+        # remote 10s < local 60s
+        client = create_standalone_client(self)
+        data = encode_transport_parameters(
+            # Note the timeout parameter here is in milliseconds.
+            QuicTransportParameters(
+                original_destination_connection_id=client.original_destination_connection_id,
+                max_idle_timeout=10000,
+            )
+        )
+        client._parse_transport_parameters(data)
+        self.assertAlmostEqual(client._remote_max_idle_timeout, 10.0)
+        self.assertAlmostEqual(client.idle_timeout(), 10.0)
+        # remote 120s > local 60s
```

Comment:
Sole blank lines between the 3 scenarios would help readability.

Also: I know I've haven't done been good at doing this in the past, but I think all comments going forward should start with a capital and end with a dot :)